src/Discord.Net.Commands/CommandParser.cs

```diff
@@ -12,9 +13,8 @@ private enum ParserPart
             None,
             Parameter,
             QuotedParameter
-        }
-        
-        public static async Task<ParseResult> ParseArgsAsync(CommandInfo command, ICommandContext context, bool ignoreExtraArgs, IServiceProvider services, string input, int startPos)
+        }    
```

Comment:
Missing new line between enum and method

Reply:
I must have mistook "remove unnecessary whitespace" as the line instead of the space after. Fixed now.